testing/mod.ts

```diff
@@ -75,15 +103,25 @@ function createTestResults(tests: TestDefinition[]): TestResults {
   );
 }
 
+function promptTestTime(time: number = 0) {
+  let timeStr = "";
+  if (time >= 1000) {
+    timeStr = `${time / 1000}s`;
+  } else {
+    timeStr = `${time}ms`;
+  }
+  return gray(italic(`(${timeStr})`));
```

Comment:
What do you think about making this turn yellow or red if the test takes a long time?

Reply:
Jest displays any test that takes more than 5 seconds in red:

https://github.com/facebook/jest/blob/14049336beeac972b5871a2ca419df2de196dc24/packages/jest-reporters/src/get_result_header.ts#L36-L43

Reply:
Agree with that. But we have to find the good thresholds, also those thresholds has to be configurable also with args or not?

Reply:
I’d say that they shouldn’t be configurable since Jest is very popular, and so their threshold is probably pretty good.

Reply:
pushed